pages/common/sc-im.md

```diff
@@ -0,0 +1,22 @@
+# SC-IM
+
+> A curse based, vim-like spreadsheet calculator.
+
+- Move Between Cells:
+
+`h, j, k, l, arrow keys`
+
+- Enter a string on the current cell:
```

Comment:
Perhaps `in` instead of `on` would read better here?

Also, perhaps you could add some examples above this one of how to start `sc-im`?

Reply:
Fixed: Changed to *into*